TARGETS

```diff
@@ -25,6 +25,14 @@ rocksdb_compiler_flags = [
     "-DTBB",
     # Needed to compile in fbcode
     "-Wno-expansion-to-defined",
+    # Added missing flags from output of build_detect_platform
```

Comment:
will all these flags compatible with fbcode? The compile output will be linked against other project in fbcode that don't have these flags.

Reply:
How do I find that out?  cc @maysamyabandeh

Reply:
I think we can remove `-std=c++11` and `-march=native`. The rest are pretty safe to me (enable rocksdb macros and warning flags)

Reply:
updated, can I get an approval?

Reply:
-DHAVE_SSE42 is under "rocksdb_arch_preprocessor_flags" below in this file. I guess pclmul should go there too as it's also x86-specific. BTW, maybe syncup with @siying as he has a pending PR for this file as well (#3533)

Reply:
So these 2 flags should be removed from this block? cc @siying

Reply:
I guess so. I just followed the previous practice. I don't have the context why it was like that in the first place. If I would guess, I would guess the same reason as what you would guess.